src/Query/QueryBuilder.php

```diff
@@ -585,7 +586,7 @@ public function getMaxResults()
     /**
      * Either appends to or replaces a single, generic query part.
      *
-     * The available parts are: 'select', 'from', 'set', 'where',
```

Comment:
What's the reason for this change? What happens to existing apps that call `$qb->add('set', …)`?